mslib/utils/airdata.py

```diff
@@ -145,7 +141,7 @@ def update_airspace(force_download=False, countries=["de"]):
         if (force_download or is_outdated or not file_exists) \
                 and QtWidgets.QMessageBox.question(
                     None, "Allow download",
-                    f"The selected {country} airspace needs to be downloaded ({data[-1]})"
+                    f"The selected {country} airspace needs to be downloaded ({round(int(data[-1]) / (1024**2), 3)}MB)"
```

Comment:
have a look on this ;) 
```
import humanfriendly
humanfriendly.format_size(NUMBER)
```
more examples https://pypi.org/project/humanfriendly/#description

we have this in our stack.

Reply:
Done